extensions/sql-bindings/package.json

```diff
@@ -65,7 +65,8 @@
     "fast-glob": "^3.2.7",
     "jsonc-parser": "^2.3.1",
     "promisify-child-process": "^3.1.1",
-    "vscode-nls": "^4.1.2"
+    "vscode-nls": "^4.1.2",
+    "vscode-languageclient": "5.2.1"
```

Comment:
I'm not seeing this be something that's used at runtime so it should be able to go into devDependencies instead

Reply:
Yeah but that's only the type - not any of the actual code from the library. So you only need it during build.